tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -47,6 +48,10 @@
     import torch
 
 
+if is_pyctcdecode_available():
+    from huggingface_hub import snapshot_download
+
```

Comment:
You can remove `is_pyctcdecode_available`
and its import.

`huggingface_hub` is a core dependency, it will always be available.

Reply:
Awesome. Thanks!